### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by PranjayPathak**
- Fixed bug in id by replacing it with timestamp
- Fixed syntax issues in the template
- Fixed Task.status issue
- Minor changes in designs

### CODE REVIEW

File: src/components/TaskCell.vue [hunk 1]
```diff
@@ -20,41 +13,43 @@ export default {
     editTask() {
       this.$emit("editTask", this.task.id);
     },
-  },
-  mounted() {
-    this.task = this.taskInfo;
+    toggleStatus() {
+      this.task.status = this.task.status === "done" ? "pending" : "done";
```
Comment (ankitboghra) [inline src/components/TaskCell.vue]: since there are only 2 states possible
use boolean
i.e. task.isDone = true | false

**Comment by ankitboghra** [comment]
also remove the old `Done` code from `TaskCell.vue`

**Review by ankitboghra — CHANGES_REQUESTED** [review]

File: src/App.vue [hunk 2]
```diff
@@ -6,20 +6,24 @@ export default {
     return {
       demoTasks: [
         {
-          data: "demo task 1",
-          id: 0,
+          data: "Sample task 1",
+          id: 1646702392949,
+          status: true,
```
Comment (ankitboghra) [inline src/App.vue]: variable name can be better
ex
`isComplete`
`isFinished`

**Review by ankitboghra — APPROVED** [review]

**Comment by ankitboghra** [comment]
rest looks good

File: src/App.vue (hunk 2 above)
Comment (PranjayPathak) [reply src/App.vue]: Sure

**Pull request merged**